src/header/header-buttons.scss

```diff
@@ -0,0 +1,239 @@
+    &[role='button'] {
+      @include iui-focus;
+      cursor: pointer;
+      -webkit-tap-highlight-color: transparent;
+      transition: background-color $iui-speed-fast ease-out;
+      @media (forced-colors: active) {
+        border: 1px solid transparent;
+        border-bottom-width: 0;
+      }
+
+      // #region resets for button
+      @include iui-reset;
+      font-family: inherit;
+      background-color: unset;
+      color: unset;
+      // #endregion
+
+      &:hover {
+        background-color: rgba(var(--iui-color-foreground-body-rgb), var(--iui-opacity-6));
+      }
+    }
+  }
+}
+
+@mixin iui-header-brand-icon {
+  width: $iui-icons-large;
+  height: $iui-icons-large;
+  margin: 0 $iui-side-navigation-icon-margins;
+  fill: var(--iui-icons-color-actionable);
+  will-change: transform;
+  @media (prefers-reduced-motion: no-preference) {
+    transition: transform $iui-speed-fast ease-out;
+  }
+  @at-root .iui-slim & {
+    transform: scale(0.66666);
+  }
+}
+
+@mixin iui-header-brand-label {
+  margin-right: $iui-sm;
+  font-size: $iui-font-size-leading;
+  @media (prefers-reduced-motion: no-preference) {
+    transition: font-size $iui-speed-fast ease-out;
+  }
+  @media (max-width: 1024px) {
+    display: none;
+  }
+  @at-root .iui-slim & {
+    font-size: $iui-font-size;
+  }
+}
+
+@mixin iui-header-breadcrumb-button-container($active-color: primary) {
+  max-width: 25vw;
+  transition: background-color $iui-speed-fast ease-out;
+  height: 100%;
+  display: flex;
+  overflow: hidden;
+  position: relative;
+  color: var(--iui-text-color);
+  box-sizing: border-box;
+  @media (forced-colors: active) {
+    color: LinkText;
+    border: none;
+  }
+
+  // Hover state
+  &:hover {
+    background-color: rgba(var(--iui-color-foreground-body-rgb), var(--iui-opacity-6));
+  }
+
+  // Active state
+  &[aria-current] {
+    background-color: rgba(var(--iui-color-foreground-#{$active-color}-rgb), var(--iui-opacity-6));
+    box-shadow: 0 $iui-xxs 0 0 currentColor;
+    color: var(--iui-color-foreground-#{$active-color});
+    pointer-events: none; // Removes hover effect, reduces specificity
+    @media (forced-colors: active) {
+      color: LinkText;
+      border-bottom: 3px solid LinkText;
+    }
+  }
+
+  // Disabled state
+  &.iui-disabled,
+  &[aria-disabled='true'] {
+    cursor: not-allowed;
+    background-color: transparent;
+    color: var(--iui-text-color-placeholder);
+    pointer-events: none; // Removes hover effect
+
+    // Disabled active state
+    &[aria-current] {
+      box-shadow: none;
+      background-color: var(--iui-color-background-2);
+
+      @media (forced-colors: active) {
+        border-bottom: 3px solid GrayText;
+      }
+    }
+  }
+}
+
+@mixin iui-header-breadcrumb-button {
+  @include iui-reset;
+  @include iui-focus;
+  display: inline-flex;
+  align-items: center;
+  vertical-align: middle;
+  justify-content: center;
+  height: 100%;
+  border-radius: 0;
+  max-width: 25vw;
```

Comment:
That is [what we had set previously](https://github.com/iTwin/iTwinUI/blob/bcb65221a7f6ceadd0b9b4506686f73aa686f1d8/src/header/header.scss#L242).  Maybe it would make sense to switch to `ch` to align with [breadcrumbs](https://github.com/iTwin/iTwinUI/blob/bcb65221a7f6ceadd0b9b4506686f73aa686f1d8/src/breadcrumbs/breadcrumbs.scss#L29)?

Reply:
I dont know... Maybe we want to limit whole width of button depending on the width of the header or something..
Dont have a single opinion.
I would have expected more stuff to be reused from breadcrumbs, but I guess not really needed